tools/fuzzer-runner.js

```diff
@@ -47,7 +47,7 @@ function run({ amount = 300, fuzzBrokenAutofixes = true } = {}) {
      * a crash-only fuzzer run versus an autofix fuzzer run.
      */
     const progressBar = new ProgressBar(
-        "Fuzzing rules [:bar] :percent, :elapseds elapsed, eta :etas, errors so far: :elapsedErrors",
+        "Fuzzing rules [:bar] :percent, :elapsed elapsed, eta :etas, errors so far: :elapsedErrors",
```

Comment:
```suggestion
        "Fuzzing rules [:bar] :percent, :elapseds elapsed, eta :etas, errors so far: :elapsedErrors",
```

This was intentional, to print `s` (as short for seconds) after [`:elapsed`](https://github.com/visionmedia/node-progress#tokens). For example, `19.7s elapsed`

Reply:
Oops, sorry about that.